--- Makefile.orig	2009-08-29 13:12:00.000000000 +0200
+++ Makefile	2009-08-29 13:12:31.000000000 +0200
@@ -144,13 +144,13 @@
 	rm -f config.mak config.h x264.pc
 	rm -rf test/
 
-install: libx264.a $(SONAME)
+install: $(SONAME)
 	install -d $(DESTDIR)$(bindir) $(DESTDIR)$(includedir)
 	install -d $(DESTDIR)$(libdir) $(DESTDIR)$(libdir)/pkgconfig
 	install -m 644 x264.h $(DESTDIR)$(includedir)
-	install -m 644 libx264.a $(DESTDIR)$(libdir)
+	#install -m 644 libx264.a $(DESTDIR)$(libdir)
 	install -m 644 x264.pc $(DESTDIR)$(libdir)/pkgconfig
-	ranlib $(DESTDIR)$(libdir)/libx264.a
+	#ranlib $(DESTDIR)$(libdir)/libx264.a
 ifeq ($(SYS),MINGW)
 	$(if $(SONAME), install -m 755 $(SONAME) $(DESTDIR)$(bindir))
 else
