diff -Naur s25rttr.orig/0.8/build/postinstall.sh.cmake s25rttr/0.8/build/postinstall.sh.cmake
--- s25rttr.orig/0.8/build/postinstall.sh.cmake	2013-12-22 16:06:54.000000000 +0100
+++ s25rttr/0.8/build/postinstall.sh.cmake	2013-12-25 09:19:06.887223374 +0100
@@ -178,34 +178,6 @@
 	popd
 }
 
-mecho --blue "## Extracting debug info from files and saving them into dbg"
-
-# strip out debug symbols into external file
-if [ "$COMPILEFOR" = "apple" ] ; then
-	echo "extraction not supported ???"
-	i686-apple-darwin10-strip -S ${DESTDIR}bin/s25client
-	i686-apple-darwin10-strip -S ${DESTDIR}share/s25rttr/driver/video/libvideoSDL.dylib
-	i686-apple-darwin10-strip -S ${DESTDIR}share/s25rttr/driver/audio/libaudioSDL.dylib
-	i686-apple-darwin10-strip -S ${DESTDIR}share/s25rttr/RTTR/s25update
-	i686-apple-darwin10-strip -S ${DESTDIR}share/s25rttr/RTTR/sound-convert
-	i686-apple-darwin10-strip -S ${DESTDIR}share/s25rttr/RTTR/s-c_resample
-elif [ "$COMPILEFOR" = "windows" ] ; then
-	extract_debug_symbols s25client.exe
-	extract_debug_symbols driver/video/libvideoWinAPI.dll
-	extract_debug_symbols driver/video/libvideoSDL.dll
-	extract_debug_symbols driver/audio/libaudioSDL.dll
-	extract_debug_symbols RTTR/s25update.exe
-	extract_debug_symbols RTTR/sound-convert.exe
-	extract_debug_symbols RTTR/s-c_resample.exe
-elif [ "$COMPILEFOR" = "linux" ] ; then
-	extract_debug_symbols bin/s25client
-	extract_debug_symbols share/s25rttr/driver/video/libvideoSDL.so
-	extract_debug_symbols share/s25rttr/driver/audio/libaudioSDL.so
-	extract_debug_symbols share/s25rttr/RTTR/s25update
-	extract_debug_symbols share/s25rttr/RTTR/sound-convert
-	extract_debug_symbols share/s25rttr/RTTR/s-c_resample
-fi
-
 mecho --blue "## Performing additional tasks"
 
 # create app-bundle for apple
@@ -280,29 +252,6 @@
 	cp -v ${mingw}/bin/zlib1.dll ${DESTDIR}RTTR || exit 1
 
 	rmdir --ignore-fail-on-non-empty -v ${DESTDIR}S2
-elif [ "$COMPILEFOR" = "linux" ] ; then
-	miniupnpc=/usr/lib/libminiupnpc.so
-	case "$COMPILEARCH" in
-		i686|*86)
-			if [ ! "$(uname -m | sed s/i686/i386/g)" = "$COMPILEARCH" ] ; then
-				miniupnpc=/usr/i686-pc-linux-gnu/lib/libminiupnpc.so
-			fi
-		;;
-		x86_64|*64)
-			if [ ! "$(uname -m)" = "$COMPILEARCH" ] ; then
-				miniupnpc=/usr/x86_64-pc-linux-gnu/lib/libminiupnpc.so
-			fi
-		;;
-	esac
-
-	if [ -f $miniupnpc ] ; then
-		mkdir -p ${DESTDIR}${PREFIX}/lib/ || exit 1
-		cp -rv $miniupnpc* ${DESTDIR}${PREFIX}/lib/ || exit 1
-	else
-		echo "libminiupnpc.so not found at $miniupnpc"
-		echo "will not bundle it in your installation"
-		echo "install it from http://packages.siedler25.org by yourself"
-	fi
 fi
 
 exit 0
