--- common.mk.orig	2009-08-29 14:02:04.000000000 +0200
+++ common.mk	2009-08-29 14:02:32.000000000 +0200
@@ -477,7 +477,7 @@
 
 .PHONY: install-common install-headers install-bin install-man
 install-common: \
-  install-static-libraries install-shared-libraries \
+  install-shared-libraries \
   install-headers install-bin install-man
 
 INSTALL_LIB_CMD = $(INSTALL_DATA) $$p $(DESTDIR)$(LIBINST_DIR)/$$p
